keras_nlp/src/models/stable_diffusion_v3/t5_xxl_tokenizer.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.src.models.t5.t5_tokenizer import T5Tokenizer
+
+try:
+    import tensorflow_text as tf_text
+except ImportError:
+    tf_text = None
+
+
+class T5XXLTokenizer(T5Tokenizer):
```

Comment:
Can we just direclty use t5 tokenizer and add a packing layer that adds eos? `StartEndPacker`? That would fit with how we do this for all the other text models.

Reply:
Thanks for pointing out. Didn't know there was a `StartEndPacker`. Will give it a try.

Reply:
I'm also working on implementing `CLIPPreprocessor`. It seems like a better solution than making extensive modifications.

Reply:
`T5XXLPreprocessor` and `CLIPPreprocessor` have been implemented and their numerics have been verified.